.storybook/dbTheme.js

```diff
@@ -0,0 +1,30 @@
+import { create } from '@storybook/theming/create';
+
+const TEXT = '#131821';
+const GREY = '#FDFDFD';
+
+export default create({
+  base: 'light',
+  brandTitle: 'DB UI Elements ' + process.env.VERSION,
```

Comment:
this either doesn't seem to work on https://db-ui.github.io/elements/review/feat-storybook-theming/?path=/story/00-intro-welcome--page so far, or just hasn't been deployed so far.